content/articles/updating-spatial-database-via-flask-restful/index.md

```diff
@@ -0,0 +1,265 @@
+In this tutorial, we are going to build a REST API that can be used to update a geospatial database.
+
+Using flask restful, we will perform basic CRUD(create, read, update and delete) operations on spatial data. It is recommended that readers should know how to create a geospatial database.
+
+You can click here to learn how to create a geodatabase using flask, PostgreSQL and Postgis.
+
```

Comment:
@isaiaholadapo Looks like there's a missing link here.

Reply:
I have fixed it.